src/py42/sdk/__init__.py

```diff
@@ -199,6 +202,15 @@ def auditlogs(self):
         """
         return self._clients.auditlogs
 
+    @property
+    def cases(self):
+        """A collection of methods and properties for managing cases and cases events.
```

Comment:
`cases events` should be `case events` or `cases' events`

Reply:
Changed.